website/content/docs/other-specifications/acl-policy.mdx

```diff
@@ -382,6 +382,20 @@ The `policy` field for the plugin rule can have one of the following values:
 - `deny`: do not allow the resource to be read or modified. Deny takes
   precedence when multiple policies are associated with a token.
 
+## UI rules
```

Comment:
Taken with the rest of this doc, this header implies there's a `ui` section. Let's call this:

```suggestion
## Configuring ACLs for the web UI
```